e2e/src/main/tests/patient/patient-crud.e2e-spec.ts

```diff
@@ -63,12 +64,12 @@ describe('TC0001_PatientManagement_e2e', () => {
 		await because('All fields are evaluated as expected').expect(Promise.resolve(row[3])).toEqual(p.email);
 	}
 
-	async function checkPatient(patient: any) {
+	async function checkPatient(pPatient: any) {
```

Comment:
@atzurk , I would keep the var names as "patient" and "allergy" for clarity. The p is not needed.

Reply:
> @atzurk , I would keep the var names as "patient" and "allergy" for clarity. The p is not needed.

done, it was just to avoid the var/const shadowed error from tslint...